specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json

```diff
@@ -4829,6 +4837,18 @@
           "format": "int32",
           "description": "Specifies the fault domain of the virtual machine."
         },
+        "computerName": {
+          "type": "string",
+          "description": "The computer name assigned to the virtual machine."
```

Comment:
Is it possible to change computerName of a VMSS::VMinstance via a VMInstance PUT? if no pls mark is as readOnly.

Reply:
These properties are under VirtualMachineInstanceView which itself is ReadOnly. So we don't have to set ReadOnly on each property explicitly.